app/assets/stylesheets/assignments/calendar.scss

```diff
@@ -0,0 +1,73 @@
+#calender {
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 100%;
+  height: 100%;
+  min-width: 800px;
+  margin-bottom: 5%;
+}
+
+.cal-head {
+  text-align: right;
+  color: #999;
+  font-weight: normal;
+}
+
+.cal-cell {
+  width: 14%;
```

Comment:
With weird numbers like this (I assume it's vaguely 1/7) it's nice to explain how you arrived at them.